minigames/counting_modes.py

```
@@ -350,6 +350,7 @@ async def counting(message: discord.Message, config: dict | None = None) -> None
         new_mode = random.randint(1, len(modeMap))
         goal = get_goal(new_mode)
         embed = tanjunEmbed(
+            colour=EmbedColor.ERROR,
```

Flake8 (7.3.0)

[error] undefined name 'EmbedColor'

(F821)